player.ml

```diff
@@ -0,0 +1,35 @@
+open Card
+
+exception CardNotInHand of Card.t
+
+type t = {
+  name : string;
+  hand : Card.t list
+}
+
+let name = t.name
+
+let player_hand = t.hand
+
+let is_uno cards = 
+  match cards with
+  | [] -> true
+  | h :: [] -> true
+  | _ -> false
+
+let add_card player card = 
+  {player with hand = card::hand}
+
+(** [remove_first_of_dup hand card acc] is the player's hand after removing the
+    first of any given card form the player's original hand [hand] if the card
+    [card] is in the hand. The accumulator [acc] stores the cards checked and
```

Comment:
You don't need to add a specification comment about the accumulator. Also, I think it should be a (* not (**